docs/single_machine_testing.md

```diff
@@ -0,0 +1,63 @@
+# Single-Machine Testing
+
+This document describes the steps required to run Neurobooth on a single machine for testing purposes. There is currently no support for running a production neurobooth on a single machine.
+
+Note: You must be running Windows 10 or greater to run Neurobooth on your machine.
+
+## Configuration
+To run on a single machine you must modify the neurobooth config file, typically "neurobooth_os_config.json". 
+The following changes should be applied:
+- The entries for acquisition, control, and presentation must have name, user, and password all set to  "".
+- The entries for acquisition, control, and presentation must have port set to different values
+- TODO: Describe how to set the monitor width, etc.
+
+If you're running the db on the same machine as the neurobooth code, the following additional changes are required:
```

Comment:
~Could mention setting NB_CONFIG environment variable, which should point to your own copy of the config files.~

Could add a relative link to the system configuration doc.

Reply:
I'll add a link in the next PR for this

Reply:
actually, i just put it in this one.